### PR TITLE
MP update 0.10.3

### PR DESCRIPTION
**Pull request by atobiszei**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -20,6 +20,19 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
...
+    sha256 = "74d544d96f4a5bb630d465ca8bbcfe231e3594e5aae57e1edbf17a6eb3ca2506",
+    urls = [
+        "https://storage.googleapis.com/mirror.tensorflow.org/github.com/bazelbuild/bazel-skylib/releases/download/1.3.0/bazel-skylib-1.3.0.tar.gz",
+        "https://github.com/bazelbuild/bazel-skylib/releases/download/1.3.0/bazel-skylib-1.3.0.tar.gz",
+    ],
+)
+load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
+bazel_skylib_workspace()
+load("@bazel_skylib//lib:versions.bzl", "versions")
+versions.check(minimum_bazel_version = "3.7.2")
```
Comment (rasapala) [inline WORKSPACE:34]: Change to 6.0.0

File: WORKSPACE [hunk 2]
```diff
@@ -350,9 +357,8 @@ google_cloud_cpp_common_deps()
 
 load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
 grpc_deps()
-
-load("@com_github_grpc_grpc//bazel:grpc_extra_deps.bzl", "grpc_extra_deps")
-grpc_extra_deps()
+#load("@com_github_grpc_grpc//bazel:grpc_extra_deps.bzl", "grpc_extra_deps")
```
Comment (rasapala) [inline WORKSPACE:360]: Not needed anymore?

File: src/BUILD [hunk 3]
```diff
@@ -135,6 +135,11 @@ cc_shared_library(
         "@FP16//:__subpackages__",
         "@flatbuffers//:__subpackages__",
         "@ruy//:__subpackages__",
+
```
Comment (rasapala) [inline src/BUILD:138]: No space here.

**Review by rasapala — APPROVED** [review]
LGTM

**Review by dtrawins — APPROVED** [review]

**Pull request merged**